### ISSUE TITLE
Version changes for 3.11.2 release have not been done properly

### ISSUE THREAD

**Issue by mojito317** [editorconfig-checker/editorconfig-checker#621]
In the previous releases I see the tagged lines with `x-release-please-version` comments were updated (e.g. https://github.com/editorconfig-checker/editorconfig-checker/pull/593) but [in the most recent (3.11.2) release it did not happen](https://github.com/editorconfig-checker/editorconfig-checker/blob/v3.11.2/cmd/editorconfig-checker/main.go#L25) so when I wanted to update to the latest version, I got this error message:
```sh
Version from config file is not the same as the version of the binary
Binary: v3.11.1, Config v3.11.2
make: *** [Makefile:142: check-fmt] Error 1
Error: Process completed with exit code 2.

```

Could you please fix the release or make a new one with the latest changes that also has the version bumps in the files where it is needed? Thanks in advance!

**Comment by theoludwig** [comment]
@mojito317 Thanks for reporting, should be fixed with v4.0.0.

**Comment by klaernie** [comment]
Sorry, my bad, since this was a fix backported to v3 it was done without the regular automation, which skipped this one piece.

Since v3.11.2 was supposed to be the last v3 release, I could not leave this alone and fixed the problem properly and published v3.11.3 - which now hopefully will truly be the last v3.
